src/steps/fetch.py

```diff
@@ -79,7 +79,7 @@ def fetch_commit_sha(clone_dir):
             cwd=clone_dir
         )
         commit_log = process.stdout
-        logger.info(commit_log)  # display last commit details in log
+        logger.info(' '.join(commit_log.split()[0:2]))  # display last commit details in log
```

Comment:
maybe just output the `commit_sha` from the line below?

Reply:
you're right ... smh 😄